mod-fqm-manager/src/main/resources/corsair/mod-fqm-manager/features/entity-types.feature

```diff
@@ -131,4 +131,12 @@ Feature: Entity types
   Scenario: Refresh materialized view for tenant
     Given path '/entity-types/materialized-views/refresh'
     When method POST
-    Then status 204
\ No newline at end of file
+    Then status 204
+
+  Scenario: Ensure all entity type and field names are localized
+    Given path 'entity-types'
+    When method GET
+    Then status 200
+
+    # calls this feature on each entity type summary
+    * call read('entity-type-localized.feature') response
```

Comment:
This unintuitive line here is the best way to do a "for loop" in Karate; it will call `entity-type-localized.feature` on each element in the array `response`, taking the properties from each element (`id` and `label` in our case) and dumping them into the callee's scope.